tests/app/test_error_handlers.py

```diff
@@ -65,3 +68,22 @@ def test_jwt_validation_error(self, mock_request):
 
         self.assertEqual(response.status_code, 403)
         self.assertTrue('Not signed in'.encode() in response.data)
+
+    def test_csrf_token_expired(self):
+        app.config['WTF_CSRF_ENABLED'] = True
+        response = self.app.post('/sign-in/', data=self.sign_in_form, follow_redirects=True)
+        app.config['WTF_CSRF_ENABLED'] = False
+
+        self.assertEqual(response.status_code, 400)
+
+    def test_csrf_token_expired_on_sending_message(self):
+        app.config['WTF_CSRF_ENABLED'] = True
+        self.app.set_cookie('localhost', 'authorization', 'session_key')
+        self.patcher = patch('redis.StrictRedis.get', return_value=encoded_jwt_token)
+        self.patcher.start()
+        response = self.app.post("/secure-message/create-message/?case_id=123&ru_ref=456&survey=789",
+                                 data='{"test":"test"}', headers=self.headers, follow_redirects=True)
+        app.config['WTF_CSRF_ENABLED'] = False
+        self.patcher.stop()
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('To help protect your information we have signed you out.'.encode() in response.data)
```

Comment:
These tests contain a lot of setup and teardown that could _maybe_ go into setup and teardown functions if it doesn't jeopardise the other tests in the set

Reply:
The issue is that this there's only two test that needs `WTF_CSRF_ENABLED`. Enabling it in the setup will make all the other tests fail because they use WTForms 😞